drake/examples/Quadrotor/runDynamics.cpp

```diff
@@ -16,33 +16,68 @@ using namespace Eigen;
 int main(int argc, char* argv[]) {
   const size_t num_lidar_points = 100;
 
-  double final_time = argc >= 2 ? atof(argv[1]) : std::numeric_limits<double>::infinity();
-  cout << "Running simulation for " << final_time << " seconds." << endl;
+  SimulationOptions options = default_simulation_options;
+  options.realtime_factor = 1.0;
+  options.initial_step_size = 0.01;
```

Comment:
You're right. It does repeat the default. Removing them now. Thank for catching that!